storage/PreloaderThread.cpp

```diff
@@ -79,7 +86,8 @@ std::size_t PreloaderThread::preloadNUMAAware(
     const std::size_t num_slots) {
 #ifdef QUICKSTEP_HAVE_LIBNUMA
   std::size_t blocks_loaded = 0;
-  const NUMAPlacementScheme *placement_scheme = relation.getNUMAPlacementSchemePtr();
+  const NUMAPlacementScheme *placement_scheme =
+      relation.getNUMAPlacementSchemePtr();
```

Comment:
The intention was to be able to call the function regardless of the NUMA availability or not. If NUMA is not present the function body just becomes ``{return 0;}``, which is not really a big compilation overhead. 

Nonetheless, I have put the function in NUMA guards now and therefore the call site is also protected by NUMA guards.